src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -367,14 +382,16 @@ def get_trace_margin(self):
 
 
 ### Collision handling ###
-def _are_unit_indices_overlapping(sparsity_mask, i, j):
+def _units_are_spatially_overlapping(sparsity_mask, i, j):
```

Comment:
I think the difference is:

is the function answering a question (start with is/are)
is the function stating a fact (start with subject then do is/are)

In general I think this codebase tends toward wording these as questions rather than as statements.

Reply:
Interesting yes, I usually have thought about this in the context of how they read in conditional statements. For example:

```
if units_are_overlapping():
    ....
```
reads more naturally than:
```
if are_units_overlapping():
    ...
```

but just looking at it from the perspective of a function name, `are_units_overlapping()` gives more indication of what the function does (answering a question) than `units_are_overlapping()` (almost requires already knowing this returns a boolean to realise it is answering a question). So I'm not so strong on either approach and having a quick look
[1](https://stackoverflow.com/questions/1566745/boolean-method-naming-readability)
[2](https://stackoverflow.com/questions/47996388/python-boolean-methods-naming-convention)
[3](https://softwareengineering.stackexchange.com/questions/221868/should-methods-that-return-boolean-be-named-after-a-question-or-an-assertion)
. But maybe now is a good time to add something on this to the contributing guide or a style guide?

Reply:
I vote fot the question style: `are_units_overlapping` (?)

Reply:
I vote for the question as well. It's what the library is already doing so I think changing this midstream would be a big refactor. @JoeZiminski I would say feel free to add a note to the style guide!

Reply:
Makes sense to keep the same style of questions for booleans then. Questions it is.

Reply:
thanks all! I have reverted to question format